build-system/tasks/serve.js

```diff
@@ -25,7 +26,8 @@ var port = argv.port || process.env.PORT || 8000;
  * Starts a simple http server at the repository root
  */
 function serve() {
-  var server = gls.static(argv.path || '/', port);
+  var serverScript = path.join(__dirname, '../server.js')
+  var server = gls.new(['--harmony', serverScript, (argv.path || '/'), port]);
```

Comment:
I don't think so. I think I just got excited about the future now that I have node 5 and all that ;). I'll remove it.